examples/basics/visuals/mesh.py

```diff
@@ -52,9 +52,9 @@ def __init__(self):
         vcolor[:, 1] = np.random.normal(size=nv)
         vcolor[:, 2] = np.linspace(0, 1, nv)
         self.meshes.append(visuals.MeshVisual(verts, faces, vcolor))
-        self.meshes.append(visuals.MeshVisual(verts, faces, vcolor, 
-                                              shading='flat'))
-        self.meshes.append(visuals.MeshVisual(verts, faces, vcolor, 
+        self.meshes.append(visuals.MeshVisual(verts, faces, vcolor,
+                                              shading=' flat'))
+        self.meshes.append(visuals.MeshVisual(verts, faces, vcolor,
                                               shading='smooth'))
 
         # Lay out meshes in a grid
```

Comment:
You can remove `tr_sys` and `mesh.tr_sys` below--all visuals now come with their own `TransformSystem`.